test/metabase/analytics/stats_test.clj

```diff
@@ -222,6 +222,8 @@
 (deftest new-impl-test
   (mt/with-temp [QueryExecution _ (merge query-execution-defaults
                                          {:error "some error"})
+                 QueryExecution _ (merge query-execution-defaults
+                                         {:error "some error"})
```

Comment:
I added it so that we get 2 errors and 1 complete, it was 1:1 before and we might count it in the reverse order without knowning, so I think this is needed.